src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMs) {
   newCommentTimestampMs = timeMs;
 }
 
+/**
+ * Returns an array of the `DiscussionComment`s within
+ * `TIME_TOLERANCE_MILLISECONDS` milliseconds to the `timestampMilliseconds`.
+ *
+ * <p>This returns an empty array if no elements are nearby.
```

Comment:
This is important information and should be part of the main JSDoc, not in a new paragraph.

Reply:
Combined